Fix hyperlinks and updated package version (#58)
Azure/PyRIT: doc/setup/install_pyrit.md

pip install pyrit
```

> Next, check out our [doc](https://github.com/Azure/PyRIT/tree/main/doc) and run them in your environment!

Reviewer: ```suggestion
Next, check out our [docs](https://github.com/Azure/PyRIT/tree/main/doc) and run the notebooks in your environment!
```

> Next, check out our [docs](https://github.com/Azure/PyRIT/tree/main/doc) and run the notebooks in your environment!

Reviewer 2: prob be a relative link
